src/frequenz/channels/file_watcher.py

```diff
@@ -127,13 +128,19 @@ def __init__(
         self,
         paths: list[pathlib.Path | str],
         event_types: abc.Iterable[EventType] = frozenset(EventType),
+        *,
+        enable_polling: bool = True,
```

Comment:
If you are using a positive term then I would use the same as `watchfiles` (`force_polling`), as probably even when passing `force_polling=False` (`enable_polling=False`) polling will probably be used if I notify or other event based system calls are not available in the OS.